packages/calcite-components/src/components/table-header/resources.ts

```diff
@@ -13,3 +13,9 @@ export const CSS = {
   lastCell: "last-cell",
   staticCell: "static-cell",
 };
+
+export const ICONS = {
+  allSelected: "check-square-f",
```

Comment:
nitpick: native checkbox calls these `checked`, `indeterminate` and `unchecked`. Although what you have might be easier to understand. :)

Reply:
Updated for consistency